sql/testdata/select

```diff
@@ -66,7 +66,7 @@ statement ok
 INSERT INTO abc VALUES (0, 1, 2)
 
 query II
-SELECT a,b FROM abc WHERE CASE WHEN a != 0 THEN b/a > 1.5 ELSE false END
+SELECT a,b FROM abc WHERE CASE WHEN a != 0 THEN b::float/a::float > 1.5 ELSE false END
```

Comment:
I'm more on the fence. I think the current behavior is understandable. And I think having integer division result in an integer is also expected given the behavior of other programming languages.